.gitignore

```diff
@@ -0,0 +1 @@
+.DS_Store
```

Comment:
there are several things which should be ignored, like user based project settings.
you can use https://gitignore.io/ to generate a gitignore file which will cover those, just type in Xcode and SwiftPackageManager